Source/SIMPLib/DataContainers/DataArrayProxy.cpp

```diff
@@ -324,3 +324,13 @@ void DataArrayProxy::updatePath(DataArrayPath::RenameType renamePath)
     path = newPath.serialize();
   }
 }
+
+uint8_t DataArrayProxy::getflag()
+{
+	return flag;
+}
+
+void DataArrayProxy::setflag(uint8_t newFlag)
+{
+	flag = newFlag;
```

Comment:
Both flag methods need an indentation check, and there's no new line at the end (again, not sure if we care about that)